solution/backend/regulations/templates/regulations/about.html

```diff
@@ -1,248 +1,192 @@
 {% extends "regulations/base.html" %}
 {% load static %}
-
-{% block title_prefix %}About This Tool | {% endblock %}
-
+{% block title_prefix %}About This Tool |{% endblock title_prefix %}
 {% block header %}
```

Comment:
most if not all of the changes in this file are linting/formatting using `htmldjango.djlint` and/or `htmldjango.djhtml`